harness/determined/deploy/gcp/cli.py

```diff
@@ -185,6 +274,18 @@ def parse(s: str) -> Tuple[str, str]:
                     "optional named arguments",
                     None,
                     [
+                        Arg(
+                            "--cluster-id",
+                            type=str,
+                            default=None,
```

Comment:
is default=None needed if it's already optional same for other instances.

Reply:
Yeah, I don't think its needed. Fixed!